src/SeqCli/Cli/Features/BatchSizeFeature.cs

```diff
@@ -0,0 +1,19 @@
+﻿using SeqCli.Util;
+
+namespace SeqCli.Cli.Features
+{
+    class BatchSizeFeature: CommandFeature
+    {
+        const int DefaultBatchSize = 100;
+        int? _value;
+
+        public int Value => _value ?? DefaultBatchSize;
+
+        public override void Enable(OptionSet options)
+        {
+            options.Add("batchsize=",
+                "The number of events in each batch to send to Seq.",
```

Comment:
May sound like only N events are actually sent; could we say `"The maximum number of events to send in each request to the ingestion endpoint"?

These are also fragments, so no `.` is needed here.